python/ray/serve/backend_state.py

```diff
@@ -40,6 +40,13 @@ class ReplicaStartupStatus(Enum):
     FAILED = 4
 
 
+class GoalStatus(Enum):
+    NONE = 1
```

Comment:
just a nit but isn't this pending ?

finished reading the following part, we should add a PENDING state here

Reply:
ah yeah sure